src/conponents/Header/Header.tsx

```diff
@@ -0,0 +1,48 @@
+import cn from 'classnames';
+import React, { FormEvent } from 'react';
+import { Todo } from '../../types/Todo';
+
+interface Props {
+  todos: Todo[];
```

Comment:
Please review your code, and pass only the information you need. For instance:

```suggestion
  hasTodos: boolean;
```